plugins/discord/client/Icon.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+
+type Props = {
+  /** The size of the icon, 24px is default to match standard icons */
+  size?: number;
+  /** The color of the icon, defaults to the current text color */
+  fill?: string;
+  className?: string;
+};
+
+function DiscordLogo({ size = 24, fill = "currentColor", className }: Props) {
+  return (
```

Comment:
The icon is too big compared to others, it should be 16x16 within a 24x24 space

<img width="192" alt="image" src="https://github.com/outline/outline/assets/380914/79a5d4e2-c845-42ae-9618-739a660a8b33">